runelite-client/src/main/java/net/runelite/client/plugins/itemstats/ItemStatOverlay.java

```diff
@@ -105,7 +105,8 @@ public Dimension render(Graphics2D graphics)
 
 		int itemId = entry.getIdentifier();
 
-		if (group == WidgetInfo.EQUIPMENT.getGroupId())
+		if (group == WidgetInfo.EQUIPMENT.getGroupId() || (group == WidgetInfo.BANK_EQUIPMENT_CONTAINER.getGroupId()
+			&& widget.getParentId() == WidgetInfo.BANK_EQUIPMENT_CONTAINER.getId()))
```

Comment:
Why is this parent check necessary?

Reply:
Here is the widget hierarchy when in the bank equipment UI:
```
12.1 BANK_CONTAINER
  - 12.66 BANK_EQUIPMENT_CONTAINER
    - 12.73 Head Slot 
      - 12.73[1] item in head slot (contains ItemId)
    - 12.74 Cape Slot
    - 12.75 Neck Slot
    - 12.76 Weapon Slot
    - 12.77 Body Slot
    - 12.78 Shield Slot
    - 12.79 Legs Slot
    - 12.80 Hands Slot
    - 12.81 Feet Slot
    - 12.82 Ring Slot
    - 12.83 Ammunition Slot
```

Checking that widget.getParentID() was 12.66 seemed easier than checking is widget.getChildID() one of  (73 || 74 || 75 || 76 || 77 || 78 || 79 || 80 || 81 || 82 || 83)

Reply:
Oh, so its to prevent showing the stats over normal items in the bank? Maybe add a comment for that this seems non-obvious to me